### ISSUE TITLE
Wheels for tflite for python 3.10

### ISSUE THREAD

**Issue by rikardn** [google-coral/pycoral#58]
### Description

I don't know if this is the right place to report this, but since Python 3.10 has been released wheels for tflite would be needed.

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Relevant Log Output

_No response_</details>

**Comment by johan12345** [comment]
Looking at the [releases page](https://github.com/google-coral/pycoral/releases), Python 3.10 is now supported for the Apple M1 platform (`macosx_12_0_arm64`)... so why not the others as well?

**Comment by jakew009** [comment]
Any update on this?

**Comment by hjonnala** [comment]
Most likely tflite runtime wheels for python 3.10 would be available by end of 2022 by tensorflow/tfltie-support team. https://github.com/tensorflow/tensorflow/issues/56137

**Comment by rikardn** [comment]
This is good news!

Python 3.11 will be out before this. Do you intend to also publish wheels for Python 3.11 before the year end?

**Comment by jiapei100** [comment]
I actually build **tflite** from source and installed. However, after I successfully built and installed **pycoral** from source, I got the following **ERROR**s:

```console
➜  pycoral git:(master) ✗ python3 examples/classify_image.py \                       
--model test_data/mobilenet_v2_1.0_224_inat_bird_quant_edgetpu.tflite \
--labels test_data/inat_bird_labels.txt \
--input test_data/parrot.jpg
Traceback (most recent call last):
  File "....../pycoral/examples/classify_image.py", line 40, in <module>
    from pycoral.utils.edgetpu import make_interpreter
  File "~/.local/lib/python3.10/site-packages/pycoral/utils/edgetpu.py", line 24, in <module>
    from pycoral.pybind._pywrap_coral import GetRuntimeVersion as get_runtime_version
ModuleNotFoundError: No module named 'pycoral.pybind'
```

**Comment by hjonnala** [comment]
Please check the Linux machine wheels for tflite and pycoral for python3.10 at: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10

**Comment by rikardn** [comment]
Thanks! I have tested it and it works fine.

**Comment by rikardn** [comment]
For information:

This wheel does not work on github actions with `ubuntu-latest`: `ImportError: /lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.33' not found (required by /home/runner/work/pharmpy/pharmpy/.tox/unit/lib/python3.10/site-packages/tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.cpython-310-x86_64-linux-gnu.so)`

I also tested `ubuntu-22.04` on GHA and it works there.

**Comment by hjonnala** [comment]
Thanks for the info. It seems ubuntu-latest, on [GHA](https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/), comes with ubuntu-20.4 and likely to be change to ubuntu-22.04 soon.

**Comment by hjonnala** [comment]
> Please check the Linux machine wheels for tflite and pycoral for python3.10 at: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10

If the wheels are not working for your machine, feel free to try to build for your platform and check the issue to fix the common issues while building: https://github.com/google-coral/pycoral/issues/85#issuecomment-1308139930.. Thanks!

**Comment by ianustec** [comment]
> > Please check the Linux machine wheels for tflite and pycoral for python3.10 at: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10
> 
> If the wheels are not working for your machine, feel free to try to build for your platform and check the issue to fix the common issues while building: [#85 (comment)](https://github.com/google-coral/pycoral/issues/85#issuecomment-1308139930).. Thanks!

Hi,
I'm trying to do this (for RPI4) but the process build for python 3.7

`
make -C tensorflow/lite/tools/pip_package docker-build \                                                                        
  TENSORFLOW_TARGET=aarch64 PYTHON_VERSION=3.10
docker build -t "tflite-runtime-builder-debian-buster" --build-arg IMAGE=debian:buster .
[+] Building 1.5s (15/15) FINISHED                                                                                                                                                                      
 => [internal] load build definition from Dockerfile                                                                                                                                               0.0s
 => => transferring dockerfile: 37B                                                                                                                                                                0.0s
 => [internal] load .dockerignore                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                    0.0s
 => [internal] load metadata for docker.io/library/debian:buster                                                                                                                                   1.3s
 => [internal] load build context                                                                                                                                                                  0.0s
 => => transferring context: 39B                                                                                                                                                                   0.0s
 => [ 1/10] FROM docker.io/library/debian:buster@sha256:bb8c62f9f540a5f5f49f7ed5caae0e2889f999cd14553f2aaccaf1fa4fcb998d                                                                           0.0s
 => CACHED [ 2/10] COPY update_sources.sh /                                                                                                                                                        0.0s
 => CACHED [ 3/10] RUN /update_sources.sh                                                                                                                                                          0.0s
 => CACHED [ 4/10] RUN dpkg --add-architecture armhf                                                                                                                                               0.0s
 => CACHED [ 5/10] RUN dpkg --add-architecture arm64                                                                                                                                               0.0s
 => CACHED [ 6/10] RUN apt-get update &&     apt-get install -y       debhelper       dh-python       python-all       python-setuptools       python-wheel       python-numpy       python-pip    0.0s
 => CACHED [ 7/10] RUN pip install pip --upgrade                                                                                                                                                   0.0s
 => CACHED [ 8/10] RUN pip install pybind11                                                                                                                                                        0.0s
 => CACHED [ 9/10] RUN pip3 install pip --upgrade                                                                                                                                                  0.0s
 => CACHED [10/10] RUN pip3 install pybind11                                                                                                                                                       0.0s
 => exporting to image                                                                                                                                                                             0.0s
 => => exporting layers                                                                                                                                                                            0.0s
 => => writing image sha256:a995ffa6f8be4cb20a68448b69e28290a0a3fd6d75a1fec1362e8a60f856cb5d                                                                                                       0.0s
 => => naming to docker.io/library/tflite-runtime-builder-debian-buster                                                                                                                            0.0s
mkdir -p /Users/thomasbenedetti/Downloads/tensorflow/tensorflow/lite/tools/pip_package/out/python3/debian-buster
docker run --user 501:20 \
		--rm --interactive --tty \
		--env "PYTHON=python3" \
		--env "TENSORFLOW_TARGET=aarch64" \
		--env "BUILD_DEB=n" \
		--env "VERSION_SUFFIX=" \
		--volume /Users/thomasbenedetti/Downloads/tensorflow/tensorflow/lite/tools/pip_package/../../../..:/tensorflow \
		--volume /Users/thomasbenedetti/Downloads/tensorflow/tensorflow/lite/tools/pip_package/out/python3/debian-buster:/out \
		"tflite-runtime-builder-debian-buster" \
		/bin/bash -c "/tensorflow/tensorflow/lite/tools/pip_package/build_pip_package.sh && \
		              (cp /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/*.deb \
		                  /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3/dist/{*.whl,*.tar.gz} \
		                  /out 2>/dev/null || true)"
+++ dirname /tensorflow/tensorflow/lite/tools/pip_package/build_pip_package.sh
++ cd /tensorflow/tensorflow/lite/tools/pip_package
++ pwd
+ SCRIPT_DIR=/tensorflow/tensorflow/lite/tools/pip_package
+ PYTHON=python3
+ VERSION_SUFFIX=
+ export TENSORFLOW_DIR=/tensorflow/tensorflow/lite/tools/pip_package/../../../..
+ TENSORFLOW_DIR=/tensorflow/tensorflow/lite/tools/pip_package/../../../..
+ TENSORFLOW_LITE_DIR=/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite
++ grep '_VERSION = ' /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/tools/pip_package/setup.py
++ cut -d= -f2
++ sed 's/[ '\''-]//g'
+ TENSORFLOW_VERSION=2.5.2
+ export PACKAGE_VERSION=2.5.2
+ PACKAGE_VERSION=2.5.2
+ BUILD_DIR=/tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3
+ rm -rf /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3
+ mkdir -p /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3/tflite_runtime
+ cp -r /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package/debian /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package/setup.py /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package/MANIFEST.in /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/python/interpreter_wrapper /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3
+ cp /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/python/interpreter.py /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/python/metrics_interface.py /tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/python/metrics_portable.py /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3/tflite_runtime
+ echo '__version__ = '\''2.5.2'\'''
++ git -C /tensorflow/tensorflow/lite/tools/pip_package/../../../.. describe
fatal: detected dubious ownership in repository at '/tensorflow'
To add an exception for this directory, call:

	git config --global --add safe.directory /tensorflow
+ echo '__git_version__ = '\'''\'''
+ cd /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3
+ case "${TENSORFLOW_TARGET}" in
+ python3 setup.py bdist --plat-name=linux-aarch64 bdist_wheel --plat-name=linux-aarch64
tensorflow/lite/tools/make/Makefile:358: warning: overriding recipe for target '/tensorflow/tensorflow/lite/tools/make/gen/linux_aarch64/lib/libtensorflow-lite.a'
tensorflow/lite/tools/make/Makefile:355: warning: ignoring old recipe for target '/tensorflow/tensorflow/lite/tools/make/gen/linux_aarch64/lib/libtensorflow-lite.a'
running bdist
running bdist_dumb
running build
running build_py
running build_ext
make: Entering directory '/tensorflow'
tensorflow/lite/tools/make/Makefile:358: warning: overriding recipe for target '/tensorflow/tensorflow/lite/tools/make/gen/linux_aarch64/lib/libtensorflow-lite.a'
tensorflow/lite/tools/make/Makefile:355: warning: ignoring old recipe for target '/tensorflow/tensorflow/lite/tools/make/gen/linux_aarch64/lib/libtensorflow-lite.a'
make: Nothing to be done for 'all'.
make: Leaving directory '/tensorflow'
building 'tflite_runtime._pywrap_tensorflow_interpreter_wrapper' extension
creating build
creating build/temp.linux-x86_64-3.7
creating build/temp.linux-x86_64-3.7/interpreter_wrapper
aarch64-linux-gnu-gcc -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/tensorflow/tensorflow/lite/tools/pip_package/../../../.. -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package -I/usr/lib/python3/dist-packages/numpy/core/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/flatbuffers/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/absl -I/usr/local/lib/python3.7/dist-packages/pybind11/include -I/usr/include/python3.7m -c interpreter_wrapper/interpreter_wrapper.cc -o build/temp.linux-x86_64-3.7/interpreter_wrapper/interpreter_wrapper.o --std=c++11
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘std::__cxx11::string tflite::interpreter_wrapper::InterpreterWrapper::TensorName(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:335:26: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (!interpreter_ || i >= interpreter_->tensors_size() || i < 0) {
                        ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorType(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:345:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorSize(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:363:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorSizeSignature(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:378:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorSparsityParameters(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:398:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorQuantization(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:409:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::TensorQuantizationParameters(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:416:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::SetTensor(int, PyObject*)’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:450:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::NodeInputs(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:56:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->nodes_size() || i < 0) {         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:529:3: note: in expansion of macro ‘TFLITE_PY_NODES_BOUNDS_CHECK’
   TFLITE_PY_NODES_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘PyObject* tflite::interpreter_wrapper::InterpreterWrapper::NodeOutputs(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:56:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->nodes_size() || i < 0) {         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:539:3: note: in expansion of macro ‘TFLITE_PY_NODES_BOUNDS_CHECK’
   TFLITE_PY_NODES_BOUNDS_CHECK(i);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In member function ‘std::__cxx11::string tflite::interpreter_wrapper::InterpreterWrapper::NodeName(int) const’:
interpreter_wrapper/interpreter_wrapper.cc:548:26: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (!interpreter_ || i >= interpreter_->nodes_size() || i < 0) {
                        ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interpreter_wrapper/interpreter_wrapper.cc: In function ‘PyObject* tflite::interpreter_wrapper::{anonymous}::CheckGetTensorArgs(tflite::Interpreter*, int, TfLiteTensor**, int*)’:
interpreter_wrapper/interpreter_wrapper.cc:48:9: warning: comparison of integer expressions of different signedness: ‘int’ and ‘size_t’ {aka ‘long unsigned int’} [-Wsign-compare]
   if (i >= interpreter_->tensors_size() || i < 0) {                         \
         ^
interpreter_wrapper/interpreter_wrapper.cc:573:3: note: in expansion of macro ‘TFLITE_PY_TENSOR_BOUNDS_CHECK’
   TFLITE_PY_TENSOR_BOUNDS_CHECK(tensor_index);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
aarch64-linux-gnu-gcc -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/tensorflow/tensorflow/lite/tools/pip_package/../../../.. -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package -I/usr/lib/python3/dist-packages/numpy/core/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/flatbuffers/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/absl -I/usr/local/lib/python3.7/dist-packages/pybind11/include -I/usr/include/python3.7m -c interpreter_wrapper/interpreter_wrapper_pybind11.cc -o build/temp.linux-x86_64-3.7/interpreter_wrapper/interpreter_wrapper_pybind11.o --std=c++11
aarch64-linux-gnu-gcc -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/tensorflow/tensorflow/lite/tools/pip_package/../../../.. -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package -I/usr/lib/python3/dist-packages/numpy/core/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/flatbuffers/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/absl -I/usr/local/lib/python3.7/dist-packages/pybind11/include -I/usr/include/python3.7m -c interpreter_wrapper/numpy.cc -o build/temp.linux-x86_64-3.7/interpreter_wrapper/numpy.o --std=c++11
aarch64-linux-gnu-gcc -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/tensorflow/tensorflow/lite/tools/pip_package/../../../.. -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package -I/usr/lib/python3/dist-packages/numpy/core/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/flatbuffers/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/absl -I/usr/local/lib/python3.7/dist-packages/pybind11/include -I/usr/include/python3.7m -c interpreter_wrapper/python_error_reporter.cc -o build/temp.linux-x86_64-3.7/interpreter_wrapper/python_error_reporter.o --std=c++11
aarch64-linux-gnu-gcc -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/tensorflow/tensorflow/lite/tools/pip_package/../../../.. -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/pip_package -I/usr/lib/python3/dist-packages/numpy/core/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/flatbuffers/include -I/tensorflow/tensorflow/lite/tools/pip_package/../../../../tensorflow/lite/tools/make/downloads/absl -I/usr/local/lib/python3.7/dist-packages/pybind11/include -I/usr/include/python3.7m -c interpreter_wrapper/python_utils.cc -o build/temp.linux-x86_64-3.7/interpreter_wrapper/python_utils.o --std=c++11
creating build/lib.linux-x86_64-3.7
creating build/lib.linux-x86_64-3.7/tflite_runtime
aarch64-linux-gnu-g++ -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,-z,relro -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 build/temp.linux-x86_64-3.7/interpreter_wrapper/interpreter_wrapper.o build/temp.linux-x86_64-3.7/interpreter_wrapper/interpreter_wrapper_pybind11.o build/temp.linux-x86_64-3.7/interpreter_wrapper/numpy.o build/temp.linux-x86_64-3.7/interpreter_wrapper/python_error_reporter.o build/temp.linux-x86_64-3.7/interpreter_wrapper/python_utils.o -L/tensorflow/tensorflow/lite/tools/make/gen/linux_aarch64/lib/ -ltensorflow-lite -o build/lib.linux-x86_64-3.7/tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.so
copying tflite_runtime/__init__.py -> build/lib.linux-x86_64-3.7/tflite_runtime
copying tflite_runtime/interpreter.py -> build/lib.linux-x86_64-3.7/tflite_runtime
copying tflite_runtime/metrics_interface.py -> build/lib.linux-x86_64-3.7/tflite_runtime
copying tflite_runtime/metrics_portable.py -> build/lib.linux-x86_64-3.7/tflite_runtime
running egg_info
creating tflite_runtime.egg-info
writing tflite_runtime.egg-info/PKG-INFO
writing dependency_links to tflite_runtime.egg-info/dependency_links.txt
writing requirements to tflite_runtime.egg-info/requires.txt
writing top-level names to tflite_runtime.egg-info/top_level.txt
writing manifest file 'tflite_runtime.egg-info/SOURCES.txt'
reading manifest file 'tflite_runtime.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'tflite_runtime.egg-info/SOURCES.txt'
installing to build/bdist.linux-aarch64/dumb
running install
running install_lib
creating build/bdist.linux-aarch64
creating build/bdist.linux-aarch64/dumb
creating build/bdist.linux-aarch64/dumb/usr
creating build/bdist.linux-aarch64/dumb/usr/local
creating build/bdist.linux-aarch64/dumb/usr/local/lib
creating build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7
creating build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages
creating build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/__init__.py -> build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.so -> build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/interpreter.py -> build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/metrics_interface.py -> build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/metrics_portable.py -> build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime
byte-compiling build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime/interpreter.py to interpreter.cpython-37.pyc
byte-compiling build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime/metrics_interface.py to metrics_interface.cpython-37.pyc
byte-compiling build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime/metrics_portable.py to metrics_portable.cpython-37.pyc
running install_egg_info
Copying tflite_runtime.egg-info to build/bdist.linux-aarch64/dumb/usr/local/lib/python3.7/dist-packages/tflite_runtime-2.5.2.egg-info
running install_scripts
creating /tensorflow/tensorflow/lite/tools/pip_package/gen/tflite_pip/python3/dist
Creating tar archive
removing 'build/bdist.linux-aarch64/dumb' (and everything under it)
running bdist_wheel
installing to build/bdist.linux-aarch64/wheel
running install
running install_lib
creating build/bdist.linux-aarch64/wheel
creating build/bdist.linux-aarch64/wheel/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/__init__.py -> build/bdist.linux-aarch64/wheel/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.so -> build/bdist.linux-aarch64/wheel/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/interpreter.py -> build/bdist.linux-aarch64/wheel/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/metrics_interface.py -> build/bdist.linux-aarch64/wheel/tflite_runtime
copying build/lib.linux-x86_64-3.7/tflite_runtime/metrics_portable.py -> build/bdist.linux-aarch64/wheel/tflite_runtime
running install_egg_info
Copying tflite_runtime.egg-info to build/bdist.linux-aarch64/wheel/tflite_runtime-2.5.2.egg-info
running install_scripts
creating build/bdist.linux-aarch64/wheel/tflite_runtime-2.5.2.dist-info/WHEEL
creating 'dist/tflite_runtime-2.5.2-cp37-cp37m-linux_aarch64.whl' and adding 'build/bdist.linux-aarch64/wheel' to it
adding 'tflite_runtime/__init__.py'
adding 'tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.so'
adding 'tflite_runtime/interpreter.py'
adding 'tflite_runtime/metrics_interface.py'
adding 'tflite_runtime/metrics_portable.py'
adding 'tflite_runtime-2.5.2.dist-info/METADATA'
adding 'tflite_runtime-2.5.2.dist-info/WHEEL'
adding 'tflite_runtime-2.5.2.dist-info/top_level.txt'
adding 'tflite_runtime-2.5.2.dist-info/RECORD'
removing build/bdist.linux-aarch64/wheel
+ [[ n != \y ]]
+ exit 0
`

**Comment by abuzarra** [comment]
> Most likely tflite runtime wheels for python 3.10 would be available by end of 2022 by tensorflow/tfltie-support team. [tensorflow/tensorflow#56137](https://github.com/tensorflow/tensorflow/issues/56137)

Any update about the availability of wheels for Python 3.10?

**Comment by rikardn** [comment]
At least a wheel for linux on x86 can be found here: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10

**Comment by abuzarra** [comment]
> At least a wheel for linux on x86 can be found here: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10

I saw that wheels, I'm looking for the Linux wheels on aarch64 and  armv7l

**Comment by oberluz** [comment]
> > At least a wheel for linux on x86 can be found here: https://github.com/hjonnala/snippets/tree/main/wheels/python3.10
> 
> I saw that wheels, I'm looking for the Linux wheels on aarch64 and armv7l

Me too. When could we have official wheels for x64, aarch64 and armv7l for python 3.10?

**Comment by vinzbarbuto** [comment]
Hello everyone, I'm reaching out for an update regarding the official wheels. Currently, I urgently require Python 3.10 to integrate with an external framework. However, I'm encountering a roadblock with pycoral on my Dev Board, which operates on Python 3.7. To address this, I installed Python 3.10, but unfortunately, I'm facing difficulties in downloading the pycoral API as it seems to be unsupported by Python 3.10 at the moment.

I attempted to install pycoral using the provided link for the wheels. Despite successful installation, I encountered the following error message: `ImportError: /lib/aarch64-linux-gnu/libc.so.6: version GLIBC_2.32' not found`. Upon investigating further by executing `ldd --version` on my Coral Board, it indicates `ldd (Debian GLIBC 2.28-10) 2.28`.

Does anyone have any insights or suggestions on how to overcome these challenges?
